pennylane/measure.py

```diff
@@ -18,14 +18,8 @@
 and measurement samples.
 """
 import pennylane as qml
-from .operation import Observable, Sample, Variance, Expectation, Probability, Tensor
-from .qnodes import QuantumFunctionError
-
-
-def _remove_if_in_queue(op):
-    r"""Helper function to handle removing ops from the QNode queue"""
-    if op in qml._current_context.queue:
-        qml._current_context.queue.remove(op)
+from pennylane.operation import Observable, Sample, Variance, Expectation, Probability, Tensor
+from pennylane.qnodes import QuantumFunctionError
```

Comment:
Was there a need to switch to absolute imports or was it more so just a minor design decision? (Looks good from my side, just curious :) )

Reply:
It made Codecov stop complaining about circular imports 😄